.github/workflows/release.yml

```diff
@@ -35,19 +36,41 @@ jobs:
       MANIFEST: ${{ secrets.S3_MANIFEST }}
       ROLE: ${{ secrets.AWS_DEVTOOLS_ROLE }}
       REGION: ${{ secrets.AWS_REGION }}
+      GH_TOKEN: ${{ github.token }}
+      SSH_AUTH_SOCK: /tmp/ssh_agent.sock
     permissions:
       id-token: write
       contents: write
     steps:
-      - name: Git clone
-        uses: actions/checkout@v4
+      - name: Configure git with the bot credentials
+        run: |
+          mkdir -p ~/.ssh
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+          ssh-agent -a $SSH_AUTH_SOCK > /dev/null
+          ssh-add - <<< "${{ secrets.NEXTMVBOT_SSH_KEY }}"
+
+          echo "${{ secrets.NEXTMVBOT_SIGNING_KEY }}" > ~/.ssh/signing.key
+          chmod 600 ~/.ssh/signing.key
+
+          git config --global user.name "nextmv-bot"
+          git config --global user.email "tech+gh-nextmv-bot@nextmv.io"
```

Comment:
Is this good practice to use this kind of a mail in a public repository? this looks like it is showing internals?
maybe I am just paranoid.